From 15dc8509202ac0814ac4592d5f37c730b2d7599a Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 15 Sep 2014 07:13:11 -0400
Subject: [virt] kvm/ppc: book3s/pr - Emulate TIR register

Message-id: <1410765214-16377-56-git-send-email-dgibson@redhat.com>
Patchwork-id: 94751
O-Subject: [PATCH 55/78] KVM: PPC: Book3S PR: Emulate TIR register
Bugzilla: 1123145 1123133 1123367
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Signed-off-by: David Gibson <dgibson@redhat.com>

commit a5948fa092a04dfd6b9ee31c99eb6896c158eb08
Author: Alexander Graf <agraf@suse.de>
Date:   Fri Apr 25 16:07:21 2014 +0200

    KVM: PPC: Book3S PR: Emulate TIR register

    In parallel to the Processor ID Register (PIR) threaded POWER8 also adds a
    Thread ID Register (TIR). Since PR KVM doesn't emulate more than one thread
    per core, we can just always expose 0 here.

    Signed-off-by: Alexander Graf <agraf@suse.de>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kvm/book3s_emulate.c b/arch/powerpc/kvm/book3s_emulate.c
index 52448ef..0a1de29 100644
--- a/arch/powerpc/kvm/book3s_emulate.c
+++ b/arch/powerpc/kvm/book3s_emulate.c
@@ -566,6 +566,7 @@ int kvmppc_core_emulate_mfspr_pr(struct kvm_vcpu *vcpu, int sprn, ulong *spr_val
  case SPRN_MMCR0:
  case SPRN_MMCR1:
  case SPRN_MMCR2:
+ case SPRN_TIR:
 #endif
   *spr_val = 0;
   break;
-- 
1.7.1